client/src/containers/IssuesTab/index.jsx

```diff
@@ -22,20 +23,35 @@ class IssuesTab extends React.Component {
   }
 
   componentDidMount() {
+    const { username, repoName, repositoryId } = this.props;
     this.props.fetchIssues({
+      username,
+      repoName,
+      repositoryId,
       filter: this.state.filter
     });
   }
 
+  countOpenIssues = () => {
+    const counter = this.props.issuesData.issues.filter(issue => issue.isOpened);
+    return counter.length;
+  };
+
+  countClosedIssues = () => {
+    const counter = this.props.issuesData.issues.filter(issue => !issue.isOpened);
+    return counter.length;
+  };
+
   render() {
     const {
-      issuesData: { loading, issues }
+      issuesData: { loading, issues },
+      match
     } = this.props;
 
-    // Mock data
-    const openIssues = 20;
-    const closedIssues = 34;
+    const openIssues = this.countOpenIssues();
```

Comment:
const openIssues = issues.reduce((issue, count) => issue.isOpened ? count + 1 : count);
const closedIssues = issues.length - openIssues;

Reply:
If we have more than two statuses, this method is not suitable but your idea is good